intelmq/bots/outputs/rpz_file/output.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
+import datetime
+from datetime import datetime
+import os
+from collections import defaultdict
+from pathlib import Path
+
+from intelmq.lib.bot import OutputBot
+
+RPZ_INDICATOR_MAP = {
+    "source.fqdn": "Intel::DOMAIN"
+}
+
+now = datetime.now()
+
+
+class RpzFileOutputBot(OutputBot):
+    _file = None
+    format_filename: bool = False
+    hierarchical_output: bool = False
+    keep_raw_field: bool = False
+    message_jsondict_as_string: bool = False
+    message_with_type: bool = False
+    single_key: bool = False
+    is_multithreadable = False
+
+    encoding_errors_mode = 'strict'
+    file: str = "/opt/intelmq/var/lib/bots/file-output/rpz"
+
+    cname: str = ""
+    organization_name: str = ''
+    rpz_domain: str = ''
+    hostmaster_rpz_domain: str = ''
+    rpz_email: str = ''
+    ttl: str = '3600'
+    generate_time: str = now.strftime("%Y-%m-%d %H:%M:%S")
+    ncachttl: str = '60'
+    serial: str = now.strftime("%y%m%d%H%M")
+    refresh: str = '60'
+    retry: str = '60'
+    expire: str = '432000'
+    test_domain: str = ''
+
+    rpz_header = ""
```

Comment:
is that a configuration parameter?

Reply:
configuration parameter:

expire: str = '432000'
test_domain: str = ''

Reply:
If not, then let's mark it as private member
```suggestion
    _rpz_header = ""
```